integrations/logs/metadata.yaml

```diff
@@ -36,3 +36,40 @@
     prerequisites:
       description: |
         - A Netdata Cloud account
+- id: "windows-event-logs"
+  meta:
+    name: "Windows Event Logs"
+    link: "https://github.com/netdata/netdata/blob/master/src/collectors/windows-events.plugin/README.md"
+    categories:
+      - logs
+      - data-collection.windows-systems
+    icon_filename: "windows.svg"
+  keywords:
+    - windows
+    - windows events
+    - logs
+  overview:
+    description: |
+      The Windows Events plugin by Netdata makes viewing, exploring and analyzing Windows Events simple and
+      efficient.
+
+      The plugin automatically detects all the available channels and offers a list of "Event Channels".
+
+      By default, it aggregates events from all event channels, providing a unified view of all events.
+    visualization:
+      description: |
+        You can start exploring Windows event logs on the "Logs" tab of the Netdata UI.
+    key_features:
+      description: |
+        - Supports **Windows Event Logs (WEL)**.
+        - Supports **Event Tracing for Windows (ETW)** and **TraceLogging (TL)**, when events are routed to Event Log.
+        - Allows filtering on all System Events fields.
+        - Allows **full text search** (`grep`) on all System and User fields.
+        - Provides a **histogram** for log entries over time, with a break down per field-value, for any System Event field and any
+          time-frame.
+        - Supports coloring log entries based on severity.
+        - In PLAY mode it "tails" all the Events, showing new log entries immediately after they are received.
+  setup:
+    prerequisites:
+      description: |
+        - A Netdata Cloud account
```

Comment:
```suggestion
        - Netdata Cloud paid subscription
```

Reply:
I am adding a suggestion above for the previous integration too